components/prompt-form.tsx

```diff
@@ -69,7 +69,7 @@ export function PromptForm({
           rows={1}
           value={input}
           onChange={e => setInput(e.target.value)}
-          placeholder="Send a message."
+          placeholder="Type your message..."
```

Comment:
we typically use … the unicode character that is an ellipsis, and not three dots.